shacl12-ui/index.html

```diff
@@ -815,29 +815,29 @@ <h4>shui:AutoCompleteEditor</h4>
           <p>
             <b>Score:</b>
             1 if the value is an IRI.
```

Comment:
If the line breaks make enough difference to understanding the `.html` source to be manually applied therein, they're valuable enough to manually apply for browser rendering.
```suggestion
            <b>Score:</b><br>
            1 if the value is an IRI.<br>
```

Reply:
The lines below *Score:* are now a list, which enforces line breaks.